docs/decisions/0002-keep-static-ucr-configurations-in-memory.md

```diff
@@ -0,0 +1,51 @@
+# 2. Keep static UCR configurations in memory
```

Comment:
+1 for tracking architecture decisions in source control. But what should the threshold for this be? This feels pretty small uncontroversial to me

Reply:
Yea not sure but I found myself writing a shortened version of this in the class docstrings to I thought it would be better to have it here with a little more detail.